src/pudl/transform/sec10k.py

```diff
@@ -408,55 +620,13 @@ def core_sec10k__changelog_company_name(
     return name_changelog
 
 
-@dg.asset(
-    io_manager_key="pudl_io_manager",
-    group_name="core_sec10k",
-)
-def core_sec10k__parents_and_subsidiaries(
```

Comment:
I made a comment about this elsewhere as well but I don't think it makes sense to have this core table anymore, as the `out_sec10k__parents_and_subsidiaries` table is basically an output version of the `core_sec10k__quarterly_exhibit_21_company_ownership` table.